railties/lib/rails/application/finisher.rb

```diff
@@ -33,6 +33,14 @@ module Finisher
         end
       end
 
+      # Setup default session store if not already set in config/application.rb
+      initializer :setup_default_session_store, before: :build_middleware_stack do |app|
+        unless app.config.session_store
```

Comment:
@kaspth Because `app.config.session_store ` returns `nil` when it is set to `disabled`. 

There is no way as of now to see if the session store is not set without touching internal instance variables.